README.md

```diff
@@ -1,2 +1,23 @@
-# coldfront-plugin-api
-Simple REST API for ColdFront
+# Coldfront API Plugin
+REST API plugin for ColdFront.
+
+See `openapi.yaml` for OpenAPI specification.
+
+ColdFront doesn't currently provide a mechanism for allowing out of tree
+plugins to expose URLs, so applying the patch file at
+`patches/01_add_api_urls.patch` is required.
+
+The plugin can be enabled by adding `coldfront_plugin_api` to `ENABLED_APPS`
+in the Django `local_settings.py`.
+
+If the environment variable `PLUGIN_AUTH_OIDC` is detected, authentication
+will be done through `mozilla-django-oidc`, using the same configuration
+as the rest of ColdFront.
+
+**Note**: If using service accounts and Keycloak, it is necessary to add
+`openid` to the client scope of the service account performing the API
+request. This is because  the `mozilla-django-oidc` Django Rest Framework
```

Comment:
Is this a change that needs to be made in Keycloak? We should clarify that here.

Reply:
It's a change that needs to be made through the Keycloak dashboard when configuring the service account. https://access.redhat.com/documentation/en-us/red_hat_single_sign-on_continuous_delivery/2/html/server_administration_guide/clients#client_scopes

Reply:
Updated to include link to the above documentation.